pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -282,6 +368,13 @@ def optimizer_step(self, optimizer: torch.optim.Optimizer, lambda_closure: Calla
         # internally, the engine has a reference to the optimizer already.
         self.model.step(**kwargs)
 
+    def _handle_gradient_accumulation_steps(self):
+        if self.config.get("gradient_accumulation_steps") > 1:
```

Comment:
@tchaton could we add some comments about what's happening here and why?

Reply:
Done !